conf/config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
 	},
 	webpackDevServer: {
 		port: 3000,
-		https: true,
+		server: 'https',
```

Comment:
@petrKavulok This is cool, but it should go to wiki as well, since I am pretty sure, that I will forget to that change caused by webpack update and will struggle with https connection in dev envs. So if you can please add it there as well

Reply:
I'll add it to wiki